website/client/src/pages/private-messages.vue

```diff
@@ -722,15 +748,9 @@ export default {
     },
     filtersConversations () {
       // Vue-subscribe to changes
-      const subscribeToUpdate = this.updateConversationsCounter > -1;
-
-      const filtered = subscribeToUpdate && !this.search
-        ? this.conversations
 
-        /* eslint-disable max-len */
-        : filter(this.conversations, conversation => conversation.name.toLowerCase().indexOf(this.search.toLowerCase()) !== -1);
-
-      const ordered = orderBy(filtered, [o => moment(o.date).toDate()], ['desc']);
+      const ordered = orderBy(this.conversations,
+        [o => moment(o.date).toDate()], ['desc']);
```

Comment:
parsing string to moment to date really needed? using the normal string to order would probably sort the same?